### PR TITLE
Remove duplicate file id 

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/PS-1646

### CODE REVIEW

**Comment by pivnicek** [comment]
@odinuv I can't click you as reviewer because it's originally your PR, but this is ready...

File: tests/Functional/DownloadFilesAbsWorkspaceTest.php [hunk 1]
```diff
@@ -296,70 +302,65 @@ public function testReadFilesEsQueryFilterRunId()
...
-        $blobClient = BlobRestProxy::createBlobService($this->workspaceCredentials['connectionString']);
         try {
-            $this->assertEmpty($blobClient->getBlob(
+            $this->blobClient->getBlob(
                 $this->workspaceCredentials['container'],
-                "/download/upload/" . $id1
-            ));
+                "download/upload/" . $id1
+            );
+            $this->fail('should have thrown 404');
```
Comment (odinuv) [inline tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: you should've changed this to expectException / expectExceptionMessage

**Comment by odinuv** [comment]
yeah, had a look through it - i think it's ok - so I set you as a reviewer :)

File: tests/Functional/DownloadFilesAbsWorkspaceTest.php (hunk 1 above)
Comment (pivnicek) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: ok, yeah, you're right

**Comment by pivnicek** [comment]
:-) thanks.  I'll just change thoae fails to expected exceptions

File: tests/Functional/DownloadFilesAbsWorkspaceTest.php (hunk 1 above)
Comment (odinuv) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: no i'm not :)

Comment (odinuv) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: scratch that, there are multiple try-catch in one test, can't use expectException, it has to stay as it is

Comment (pivnicek) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php]: you can only use one expected exception per test method?  I didn't know that

Comment (odinuv) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php]: it raises an exception and terminates the code, that's the point of the exception and expectexception asserts that it had happened.

Comment (pivnicek) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php]: hmm, the latest version (that uses `expectExceptionCode` multiple times) seems to work ...

Comment (odinuv) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php]: it raises an exception and terminates the code, that's the point of the exception and expectexception asserts only that it had happened, it does not alter the flow

Comment (pivnicek) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php]: aha!  ok then, I'll revert it

Comment (odinuv) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: Yeah, the test works as long as it throws the first exception, but the rest of the code is technically unreachable

Comment (pivnicek) [reply tests/Functional/DownloadFilesAbsWorkspaceTest.php:310]: right, that makes sense.  thanks O

**Review by pivnicek — APPROVED** [review]

**Pull request merged**